definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json

```diff
@@ -965,6 +965,41 @@
     "UserRole": "ctsc-team-leader",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "24/11/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "pathfinder",
+    "UserRole": "caseworker-privatelaw-courtadmin",
```

Comment:
This is a collection field, please check if D is needeed in any scenario?

Reply:
Hi @shashisk97, as requested by @khurshedk , I am commenting here,
@khurshedk , this PR should have been pointing to https://github.com/hmcts/prl-ccd-definitions/pull/1645 then it wouldnot have been shown as your change. 
@shashisk97 , my PR has been reviewed and approved by @yogendraU . Point in concern is still in discussion between service team and BA. Ideally any change to this should be a new ticket.